redpanda/values.yaml

```diff
@@ -50,8 +50,10 @@ config:
       # Metrics are served off of this port at /metrics.
       port: 9644
     kafka_api:
```

Comment:
Suggestion: Maybe comment for the user that only first port number will be taken into forming kubernetes service. WDYT?

Reply:
Maybe I should search for internal, and document that as special, but first entry is just as clear and easier to implement!

Reply:
In hindsight, maybe it would be even better to take the list here, and generate listeners for them all. (For example, #16) - but it's tricky to tie everything together such as what service type, without making this incompatible with the actual config file format. I think I'll just do your suggestion, and then add extensible listeners support at a later date, I feel it might be a bit YAGNI for now.

Reply:
For posterity:
```diff
diff --git a/redpanda/templates/statefulset.yaml b/redpanda/templates/statefulset.yaml
index d54a203..5f8d6cd 100644
--- a/redpanda/templates/statefulset.yaml
+++ b/redpanda/templates/statefulset.yaml
@@ -101,7 +101,9 @@ spec:
               --memory={{ template "redpanda.parseMemory" . }}
               --reserve-memory=0M
               --advertise-kafka-addr=internal://{{ template "redpanda.kafka.internal.advertise.address" . }}:{{ template "redpanda.kafka.internal.advertise.port" . }}
-              --kafka-addr=internal://{{ template "redpanda.kafka.internal.listen.address" . }}:{{ template "redpanda.kafka.internal.listen.port" . }}
+            {{- range .Values.config.redpanda.kafka_api }}
+              --kafka-addr={{ .name }}://{{ .address }}:{{ .port }}
+            {{- end }}
               --advertise-rpc-addr={{ template "redpanda.rpc.advertise.address" . }}:{{ template "redpanda.rpc.advertise.port" . }}
               --rpc-addr={{ template "redpanda.rpc.listen.address" . }}:{{ template "redpanda.rpc.listen.port" . }}
               --
@@ -109,8 +111,10 @@ spec:
           ports:
             - containerPort: {{ .Values.config.redpanda.admin.port }}
               name: admin
-            - containerPort: {{ template "redpanda.kafka.internal.listen.port" . }}
-              name: kafka
+            {{- range .Values.config.redpanda.kafka_api }}
+            - containerPort: {{ .port }}
+              name: {{ .name }}
+            {{- end }}
             - containerPort: {{ template "redpanda.rpc.listen.port" $ }}
               name: rpc
           volumeMounts:
```
This gets difficult, since advertised has to be `addressed_kafka_api` for any of this to work, and we don't have an address to use at this point in the chart, and I don't really want to arbitrarily override the address.